package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "weatherapp-js",
+  "version": "1.0.0",
+  "description": "Practice API with Js Promises and Async/Await while Manipulating the DOM.",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack --progress --mode=production",
+    "watch": "webpack --progress --watch",
+    "server": "webpack-dev-server --open"
```

Comment:
Let's make sure we set up our webpack correctly.

The following steps seem not working:

- `npm i`
- `npm start`

Reply:
✔️